src/cf-forms/src/molecules/btn-inside-input.less

```diff
@@ -0,0 +1,25 @@
+.m-btn-inside-input {
+    position: relative;
+
+    .a-text-input {
+        box-sizing: border-box;
+        width: 100%;
+        padding-right: unit( 75px / @btn-font-size, em );
+
+        &.input__super {
+            padding-right: unit( 90px / @super-btn-font-size, em );
+        }
+    }
+
+    .a-btn {
+        .u-link__no-border();
+        position: absolute;
+        right: unit( 15px / @btn-font-size, em );
+        top: 0;
+
+        /* pa11y fails unless the background color is explicitly set */
+        &__link.a-btn__secondary {
```

Comment:
Is secondary currently being used in this situation? It's a color change only, and probably shouldn't be if it is.